### ISSUE TITLE
alcune considerazioni sulle notizie

### ISSUE THREAD

**Issue by rosencra** [giuseb/titolele#7]
la "notizia" continua a darmi grattacapi. È importante, quindi cercherei di non farne a meno, ma non è facile da organizzare.

Come ti ho detto è "notizia" giornalisticamente parlando, un fatto nuovo (oltre che interessante, di rilevanza pubblica, ecc, ecc), Provo a spiegare i vari nodi, forse riesci a trovare la chiave:
- una notizia evolve. Ieri la visita del Papa in Israele occupava tutte le aperture. Oggi il Papa dice che gli ebrei non hanno ucciso Cristo. E allora le notizie la notizia diventa un'altra, pur essendo della stessa famiglia. Ma occorre stabilire il nesso perché si sappia domani quanto ha smosso nel complesso la visita del Papa in Israele. Ovviamente sono possibili anche altre combinazioni tra notizie apparentemente slegate tra loro (appeno ho finito di scrivere questa cosa, mi sono ricordato delle _relazioni_)
- non sempre un titolo comprende una notizia.  "Siamo stanchi di pagare" è un'opinione - forzata - ricavata da varie considerazioni, non tutte necessariamente notizia. E qui secondo me si pone un problema. Difficile anche in questo caso immaginare che la notizia abbia una data di prima apparizione (proprio perché non è notizia. E tra l'altro non avrà nessun sequel)
- Più facile forse l'ipotesi del risultato di un sondaggio commissionato da un giornale: la notizia vera e propria non c'è, ma è comunque una novità prodotta dalla testata
- quale che sia il modo di catalogare le notizie, non è a me facile capire come meccanicamente si recuperano, quando dovrò associare un titolo alla notizia uscita la settimana precedente

**Comment by giuseb** [comment]
Proverei a concentrarmi sul primo punto; una proposta semplice semplice è di implementare un rapporto parent-child fra notizie.

1) "Papa in Israele" (notizia "orfana", che non discende direttamente da un'altra notizia)
2) "Il Papa dice che gli ebrei non hanno ucciso Cristo" (child of 1)
3) "Il Papa inciampa dalla scaletta al rientro a Ciampino" (child of 1)
4) "Il chirurgo del Gemelli amputa la gamba papale sbagliata" (child of 3)

In questo modo, puoi valutare l'impatto del viaggio in Israele nell'arco di generazioni successive di notizie. NB: questo schema prevede anche notizie "fratelle" e notizie "zie".

**Comment by rosencra** [comment]
l'esperienza dirà se va bene, ma intanto: io al titolo abbino due notizie (esempio, mamma e figlia?)

**Comment by giuseb** [comment]
No, solo la figlia. La mamma, si sa, può essere sempre identificata (al contrario del papà...)
